toolbox/siestabarriers/SiestaBarriers.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+########################################################################################
+# Copyright (c), The SiestaBarriers authors. All rights reserved.                       #
+#                                                                                      #
+# SiestaBarriers is hosted on GitHub at https://github.com/.................. #
+# For further information on the license, see the LICENSE.txt file                     #
+########################################################################################
+
+from __future__ import absolute_import
+from .SiestaBarriersBase import  SiestaBarriersBase
+from .barriers.vacancy_exchange  import VacancyExchange
+from .barriers.exchange import Exchange
+#from .barriers.kick import Kick
+#from .barriers.ring import Ring
+#from .barriers.manual import Manual
+
+
+#import Utils.utils_general
+
+#import .Utils.utils_siesta
+
+from .Utils.utils_siesta import *
+#import SiestaBarriers.Utils.utils_siesta
+
+
+__author__ = "Arsalan Akhatar"
+__copyright__ = "Copyright 2021, SIESTA Group"
+__version__ = "0.1"
+__maintainer__ = "Arsalan Akhtar"
+__email__ = "arsalan_akhtar@outlook.com," #+ \" miguel.pruneda@icn2.cat "
+__status__ = "Development"
+__date__ = "Janurary 30, 2021"
+
+
+class SiestaBarriers(SiestaBarriersBase):
+    """
+    """
+
+    def __init__(self,
+                 host_path = None,
+                 host_fdf_name = None,
+                 host_structure = None ,
+                 initial_relaxed_path = None,
+                 initial_relaxed_fdf_name = None,
+                 initial_structure = None,
+                 final_relaxed_path = None,
+                 final_relaxed_fdf_name = None,
+                 final_structure = None,
+                 image_direction = None ,
+                 trace_atom_initial_position = None,
```

Comment:
In particular all these keywords don't make sense to the uninitiated.

Reply:
- the image_direction will be just for Exchange Paths so i put it None...
- the relaxed one is bcz:
 first user gives the pristine  case structure and initial and final position of atoms so it will generate an unrelaxed  path for NEB from initial to final ... then there is method where (`prepare_endpoint_relax`) generate initial and final structure to be relaxed) after relaxing the endpoints user will provide the results folders of initial and final configuration and regenerate the  Relaxed initial Path for NEB and neb folder which contains the inputs.fdf and images* with *.DM.initial and *.DM.final to save the time to calculating the fixed endpoints(initial and final config...) scf ...
 so at first user just need to provide the pristine fdf and intial and final position of specie/s ...